Backend/internal/database/loginQ.go

```diff
@@ -0,0 +1,55 @@
+// TITLE: Login functions
+// Created by (Jacky: FafnirZ) (09/21)
+// Last modified by (Jacky: FafnirZ)(04/09/21)
+// # # #
+/*
+This module handles the database query and parsing of data for the login functions
+as well as some error handling
+**/
+package database
+
+import (
+	"database/sql"
+	"errors"
+	"log"
+
+	_ "github.com/lib/pq"
+)
+
+var connStr string = "host=db port=5432 user=postgres password=postgres dbname=test_db sslmode=disable"
+
+func GetCredentials(email string) (string, error) {
+	// not sure if this is necessary, might create a global struct which
+	// opens a connection and leaves it open????
+	db, err := sql.Open("postgres", connStr)
```

Comment:
instead of establishing a new database connection for each function u can move it out to the init function for this package, that way u can maintain a single connection throughout the entire package

Reply:
I tried putting it inside a init function but the syntax highlighting didnt like it, I'll retry it again later